### ISSUE TITLE
About Function provider article does not show examples for hyphenated functions. 

### ISSUE THREAD

**Issue by francisco-lafe** [MicrosoftDocs/PowerShell-Docs#9707]
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Typo in `Get-Foo` cmdlet" instead of "Typo."
- [X] **Verify Version:** If there is a mismatch between documentation and the behavior on your system, ensure that the version you are using is the same as the documentation. Check this box if they match or the issue you are reporting is not version specific.

### Links

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_function_provider?view=powershell-7.3

### Summary

Basically the title, it's inconsistent to have been told always to follow the "verb-noun" format for functions, and then when it comes to the function provider documentation, there's no such information available, which causes things like this:
![image](https://user-images.githubusercontent.com/90066435/212742644-9ced5ce2-6eba-40ea-92df-078f3b7aa37f.png)

### Details

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_function_provider?view=powershell-7.3

### Suggested Fix

Add an example to showcase the function definition, using the function provider:
https://superuser.com/questions/414036/get-definition-of-function-and-echo-the-code 

Simply as that. 
Regards,
